packages/react-components/src/Button/Button.tsx

```diff
@@ -23,6 +23,7 @@ export default function Button ({ children, className, floated, icon, isBasic =
     'data-for': triggerId,
     disabled: isDisabled,
     floated,
+    fluid,
```

Comment:
I like this. 

Small comment - can we make the prop`isFluid` (aligning with the other booleans) and then just `fluid: isFluid` here